src/xcode/ENA/ENA/Source/Services/TicketValidation/TicketValidationError.swift

```diff
@@ -25,33 +25,59 @@ enum TicketValidationError: LocalizedError {
 		switch self {
 		case .validationDecoratorDocument(let error):
 			switch error {
-			case .REST_SERVICE_ERROR(.receivedResourceError(.VD_ID_CLIENT_ERR)), .VD_ID_EMPTY_X5C, .VD_ID_NO_ATS_SIGN_KEY, .VD_ID_NO_ATS_SVC_KEY, .VD_ID_NO_ATS, .VD_ID_NO_VS_SVC_KEY, .VD_ID_NO_VS, .REST_SERVICE_ERROR(.receivedResourceError(.VD_ID_PARSE_ERR)):
+			case .VD_ID_EMPTY_X5C, .VD_ID_NO_ATS_SIGN_KEY, .VD_ID_NO_ATS_SVC_KEY, .VD_ID_NO_ATS, .VD_ID_NO_VS_SVC_KEY, .VD_ID_NO_VS:
 				return "\(serviceProviderError) (\(error))"
-			default:
-				return "\(AppStrings.TicketValidation.Error.tryAgain) (\(error))"
+			case .REST_SERVICE_ERROR(let serviceError):
+				switch serviceError {
+				case .receivedResourceError(.VD_ID_CLIENT_ERR), .receivedResourceError(.VD_ID_PARSE_ERR):
+					return "\(serviceProviderError) (\(serviceError.errorDescription))"
+				default:
+					return "\(AppStrings.TicketValidation.Error.tryAgain) (\(error))"
```

Comment:
```suggestion
					return "\(serviceProviderError) (\(serviceError))"
				default:
					return "\(AppStrings.TicketValidation.Error.tryAgain) (\(serviceError))"
```

With `CustomStringConvertible` conformance you will get the description by just using the error like this again. And we should use the unpacked error for the `default` case as well to get the nicer error message for the other error cases as well.